src/sidebar/services/api.js

```diff
@@ -128,12 +128,16 @@ function createAPICall(
           params
         );
 
+        /** @param {Param?} param */
+        // eslint-disable-next-line eqeqeq
```

Comment:
Can you help me understand why `eqeqeq` needed disabling here?

Reply:
Oh, that was from an earlier version of the commit where I had a `param != null` check. I've removed this now.